samples/Samples.StartupHook.AspNetCoreMvc31/Properties/launchSettings.json

```diff
@@ -16,6 +16,9 @@
 
         "DOTNET_STARTUP_HOOKS": "$(ProjectDir)$(OutputPath)profiler-lib\\OpenTelemetry.Instrumentation.StartupHook.dll",
 
+        "DOTNET_ADDITIONAL_DEPS": "$(ProjectDir)$(OutputPath)profiler-lib\\additionalDeps",
```

Comment:
These references seem to not work any more as 'profiler-lib' is removed. Seems this project was left out of refactoring. 
Base can be referenced now as `$(SolutionDir)bin\\tracer-home\\`